janusgraph-berkeleyje/src/test/java/org/janusgraph/graphdb/berkeleyje/BerkeleyGraphTest.java

```diff
@@ -112,19 +113,15 @@ public void testConcurrentConsistencyEnforcement() {
         //Do nothing TODO: Figure out why this is failing in BerkeleyDB!!
     }
 
+    @Disabled("Unable to run on GitHub Actions.")
```

Comment:
This test is just very Flaky test. If renew takes less then 1 MS, it will not throw an error and so the test will fail.
I think we just need to add an issue to rewrite this test later to artificially increase renew time.

Reply:
https://github.com/JanusGraph/janusgraph/issues/2304